app/src/main/java/com/sopt/androidstudy/presentation/home/viewmodels/MyViewModel.kt

```diff
@@ -3,12 +3,14 @@ package com.sopt.androidstudy.presentation.home.viewmodels
 import androidx.lifecycle.LiveData
 import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
+import com.sopt.androidstudy.data.model.UserData
 
 class MyViewModel : ViewModel() {
-    private val count = MutableLiveData(0)
+    private val userData = MutableLiveData(UserData("", ""))
 
-    fun increase() {
-        count.value = count.value?.plus(1)
+
+    fun getUserData(): LiveData<UserData> = userData
+    fun setUserData(data:UserData){
```

Comment:
```suggestion
    fun setUserData(data: UserData) {
```
오엠지,, reformat code 적용이 안된 것 같네요! 
알고 계실 것 같긴한데 코드를 전체 선택한 후, 맥 기준 option + command + L로 적용됩니다!